src/System.Private.CoreLib/shared/System/Runtime/CompilerServices/ContractHelper.cs

```diff
@@ -17,7 +17,7 @@ public static class ContractHelper
         /// event handlers sets the Cancel flag in the ContractFailedEventArgs, then the Contract class will
         /// not pop up an assert dialog box or trigger escalation policy.
         /// </summary>
-        internal static event EventHandler<ContractFailedEventArgs> InternalContractFailed; // TODO-NULLABLE: Should all events use nullable delegate types?
+        internal static event EventHandler<ContractFailedEventArgs>? InternalContractFailed;
```

Comment:
Since we didn't need a `= null!` here it suggests that events analysis might be broken for static events.

cc: @agocke @cston

Reply:
I think it's just a specific case of https://github.com/dotnet/roslyn/issues/34668.

Reply:
Ah right, it also happens to static fields.